python_configure.bzl

```diff
@@ -285,6 +285,14 @@ def _create_local_python_repository(repository_ctx):
     )
     python_import_lib_genrule = ""
 
+    # To embed python in C++, we need the linker and compiler flags required to embed the Python interpreter
```

Comment:
Minor nitpick: Move this code addition above line 286, where `python_import_lib_genrule` is assigned a value. It's associated with the code block starting on line 298 and shouldn't be split from it.